client/src/redux/sagas/commitsTestmanSaga.js

```diff
@@ -6,8 +6,15 @@ import { setTestman, setTestmanError } from '../actions';
 function* handleTestmanLoad() {
   try {
     const commits = yield select(state => state.commits);
-    const shas = commits.map(commit => commit.sha);
-    const testResults = yield call(fetchTests, shas[9], shas[0], 1);
+    const shas = [];
+    commits.forEach((commit, index) => {
+      if (index === 0) {
+        shas.push(commit.sha);
+      }
+
+      shas.push(commit.parents[0].sha);
```

Comment:
What this change is intended for? I can't figure out, for me it just skips the second sha

Reply:
In GitHub's API response every commit have a hash of it's parent commit. So, instead using own hashes, i build list of parents hashes and separately inserts first commit hash first. Further, i use this list in request to Testman's API.

Reply:
```
      let shas = [];
      if (commits.length > 0) {
        shas = [commits[0].sha, ...commits.map(commit => commit.parents[0].sha)];
      }
```

Reply:
I may be biased, but this look more straightforward to me :)